tests/phpunit/WC_Stripe_Webhook_Handler_Test.php

```diff
@@ -590,7 +601,7 @@ public function provide_test_process_payment_intent() {
 				'order locked'                   => false,
 				'payment type'                   => WC_Stripe_Payment_Methods::BOLETO,
 				'order status final'             => false,
-				'expected status'                => OrderStatus::PENDING,
+				'expected status'                => OrderStatus::PROCESSING,
```

Comment:
The `expected_status` was wrong; it should be `processing` when the intent succeeds.